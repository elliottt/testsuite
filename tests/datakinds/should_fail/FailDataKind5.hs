{-# LANGUAGE DataKinds #-}

module FailDataKind5 where

-- Int is not promotable
data kind Foo = Bar Int
