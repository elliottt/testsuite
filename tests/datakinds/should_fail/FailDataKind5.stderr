
FailDataKind5.hs:6:21:
    ‛Int’ of kind ‛*’ is not promotable
    In the kind ‛Int’
